docs/src/modules/components/JoyStartingLinksCollection.js

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Unstable_Grid2';
+import Paper from '@mui/material/Paper';
+import Typography from '@mui/material/Typography';
+import InstallDesktopRoundedIcon from '@mui/icons-material/InstallDesktopRounded';
+import DashboardRoundedIcon from '@mui/icons-material/DashboardRounded';
+import SettingsSuggestRoundedIcon from '@mui/icons-material/SettingsSuggestRounded';
+import Link from 'docs/src/modules/components/Link';
+
+const content = [
+  {
+    title: 'Installation',
+    description: 'Add Joy UI into your project with a few commands.',
+    link: '/joy-ui/getting-started/installation/',
+    icon: <InstallDesktopRoundedIcon color="primary" />,
+  },
+  {
+    title: 'Usage',
+    description: 'Learn the basics of working with Joy UI components.',
+    link: '/joy-ui/getting-started/usage/',
+    icon: <SettingsSuggestRoundedIcon color="primary" />,
+  },
+  {
+    title: 'Templates',
+    description: 'A selection of free application templates to get you started.',
```

Comment:
Rephrasing to match the verb tense of the previous two items

```suggestion
    description: 'Get started with our selection of free application templates.',
```